### PR TITLE
Clean up files have been read in workspace

### PR DESCRIPTION
**Pull request by baixinsui**
Fixed https://github.com/eclipse-xpanse/xpanse/issues/1831
Related to https://github.com/eclipse-xpanse/tofu-maker/pull/44 and https://github.com/eclipse-xpanse/terraform-boot/pull/123

### CODE REVIEW

File: modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployResultManager.java [hunk 1]
```diff
@@ -201,13 +201,23 @@ public void updateServiceOrderTaskWithDeployResult(DeployResult deployResult,
...
+                entityToUpdate.setTaskStatus(TaskStatus.SUCCESSFUL);
+            }
             entityToUpdate.setCompletedTime(OffsetDateTime.now());
         } else {
-            entityToUpdate.setTaskStatus(TaskStatus.FAILED);
             entityToUpdate.setErrorMsg(deployResult.getMessage());
-            entityToUpdate.setCompletedTime(OffsetDateTime.now());
+            // When the status is deploy_failed, the order status should be in-progress util the
+            // rollback is done.
+            if (deployerTaskStatus != DeployerTaskStatus.DEPLOY_FAILED) {
```
Comment (swaroopar) [inline modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployResultManager.java:217]: Does this code run even run while the rollback is still in-progress?

File: modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployResultManager.java [hunk 2]
```diff
@@ -201,13 +201,23 @@ public void updateServiceOrderTaskWithDeployResult(DeployResult deployResult,
...
+                entityToUpdate.setTaskStatus(TaskStatus.FAILED);
+            } else {
+                entityToUpdate.setTaskStatus(TaskStatus.SUCCESSFUL);
+            }
             entityToUpdate.setCompletedTime(OffsetDateTime.now());
         } else {
-            entityToUpdate.setTaskStatus(TaskStatus.FAILED);
             entityToUpdate.setErrorMsg(deployResult.getMessage());
-            entityToUpdate.setCompletedTime(OffsetDateTime.now());
+            // When the status is deploy_failed, the order status should be in-progress util the
```
Comment (swaroopar) [inline modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployResultManager.java:215]: we do not set anything to "in-progress" here as far as I see the code.

File: modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceorder/ServiceOrderEntity.java [hunk 3]
```diff
@@ -80,12 +80,12 @@ public class ServiceOrderEntity {
     @Convert(converter = ObjectJsonConverter.class)
     private List<DeployResource> previousDeployedResources;
 
-    @Column(name = "PREVIOUS_DEPLOYED_RESULT_PROPERTY", columnDefinition = "json")
+    @Column(name = "PREVIOUS_DEPLOYED_SERVICE_PROPERTY", columnDefinition = "json")
```
Comment (swaroopar) [inline modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceorder/ServiceOrderEntity.java:83]: can you also please check the lines 68 and 73. there is a typo there with the column name. If possible, we can fix with the same PR.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployResultManager.java (hunk 1 above)
Comment (baixinsui) [reply modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployResultManager.java:217]: Yes. The old code will set failed status first when deploy callback and then overwrite status to successful when rollback successful.

File: modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployResultManager.java (hunk 2 above)
Comment (baixinsui) [reply modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployResultManager.java:215]: I will remove the "in-progress" from content.

File: modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceorder/ServiceOrderEntity.java (hunk 3 above)
Comment (baixinsui) [reply modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceorder/ServiceOrderEntity.java:83]: Updated.

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**